assets/javascripts/discourse/templates/admin/plugins-patreon.hbs

```diff
@@ -48,6 +69,7 @@
   }}
 
   {{#if last_sync_at}}
-    <span class="last_synced">{{i18n "patreon.last_synced"}}: {{html-safe (format-date last_sync_at leaveAgo="true")}}</span>
+    <span class="last_synced">{{i18n "patreon.last_synced"}}:
+      {{html-safe (format-date last_sync_at leaveAgo="true")}}</span>
   {{/if}}
-</section>
+</section>
```

Comment:
Is removing the newline at end of the file intended?

Reply:
Yes, it's enforced by prettier-hbs on all files